### PR TITLE
Documentation and checklist updates for onboarding.

### PR DESCRIPTION
**Pull request by adunkman**
Draws upon work from existing documentation, including the [U.S. Tax Court onboarding document](https://docs.google.com/document/d/1ktys8x_XcFGCto1wxL5uUeAZlk87lOr31FRrZOuL_8g/edit) and Flexion’s [Onboarding Quick Reference Guide](https://docs.google.com/document/d/1jOd0wAsqTDBKsFeEvUj9ezvdTimTsqTwkYd6Imibl3c/edit#heading=h.jrlttmsqvst3) and [Team Working Agreement](https://docs.google.com/document/d/1AFBrxBBJD7CZoO7Go6hfTHCqqWLi00Eok9mGaCX3VGo/edit), this pull request updates the repository README, How We Work page, and adds two new issue templates for onboarding and offboarding employees. 

<details><summary>When reviewing, I suggest toggling views to "Display the rich diff" which will render the documents.</summary>

Screenshot showing rich diff toggle

</details>

### CODE REVIEW

**Comment by adunkman** [comment]
Tagging Mike and Jessica for review — we can chat this through in-person as well if preferable. 👯‍♀️

File: .github/ISSUE_TEMPLATE/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title: Onboard [Person]
+---
+
+Welcome to the team! 🎉 😎
+
+#### Access to tools
+
+- [ ] GitHub — speak with your team lead to determine the appropriate repositories and levels of access in the [ustaxcourt](https://github.com/ustaxcourt) and [flexion](https://github.com/flexion) organizations.
+- [ ] [Slack](https://ustaxcourt.slack.com) — look for important channels to join in [How We Work](./how-we-work.md).
+- [ ] [Sonarcloud](https://sonarcloud.io/organizations/flexion-github/projects)
```
Comment (kkoskelin) [inline .github/ISSUE_TEMPLATE/onboarding.md]: The team's reliance on Sonarcloud has been replaced with Deepscan.

File: README.md [hunk 2]
```diff
@@ -1,257 +1,116 @@
...
+DAWSON is in active use and is continually updated to deploy enhancements and new features.
 
-<br clear="both">
+- [Release notes](https://www.ustaxcourt.gov/release_notes.html) for DAWSON
+- [How we work](./docs/how-we-work.md) has our principles, product team, technical strategy, meetings cadence, tools, etc.
+- [Onboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=onboarding.md) or [offboard](https://github.com/ustaxcourt/ef-cms/issues/new?template=offboarding.md) a teammate
+- [Documentation](./docs/README.md) about the CI/CD setup, API, style guide, UX, code review, etc.
+- [Ongoing development documentation](./wikiwiki/README.md) such as designs, research data, user workflows, etc.
+- [Product roadmap](https://docs.google.com/document/d/1g3D1zPNQqVsWhJ6uGIVcHA04IdBHgYcOCm9roaSfFNk/edit) 🔒
+- [Goals of DAWSON](https://docs.google.com/document/d/1xCh1hbXjJItOWlQ2MH-GDDc0KWho88KKcHfUJcpHkSs/edit) 🔒
```
Comment (kkoskelin) [inline README.md:29]: This document (Goals) is not accessible to the general public, afaict.

File: README.md [hunk 3]
```diff
@@ -1,257 +1,116 @@
...
 
-The fork of this project in which the bulk of development is occurring is [Flexion’s fork](https://github.com/flexion/ef-cms), but all changes are pulled into the U.S. Tax Court’s repository every two weeks.
+### Backlog
 
-Artifacts for ongoing development such as designs, research data, user workflows etc. are located in the [wiki](https://github.com/flexion/ef-cms/wiki). See also [documentation about how we work](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/how-we-work.md): our principles, product team, technical strategy, meetings cadence, tools, etc.
+| Backlog | Location
+|---------|----------
+| User stories and feature work <br>(Primary backlog) | [GitHub issues in Flexion’s repository](https://github.com/flexion/ef-cms/issues#workspaces/flexionef-cms-5bbe4bed4b5806bc2bec65d3/board?repos=152320868); scrum board visible using the [ZenHub browser plugin](https://www.zenhub.com/extension).
+| Refactoring and bugs | [Trello cards in Flexion’s OpEx/DevEx board](https://trello.com/b/9tgrIFfA/ef-cms-opex-devex). 🔒
```
Comment (kkoskelin) [inline README.md:36]: This trello board is also not visible to the general public from what I can see.

**Review by kkoskelin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/onboarding.md (hunk 1 above)
Comment (adunkman) [reply .github/ISSUE_TEMPLATE/onboarding.md]: Exciting! I see [Sonarcloud is still referenced quite a bit](https://github.com/ustaxcourt/ef-cms/search?q=Sonarcloud) — has it been fully deprecated and removed, or are we still in the process of replacing it? 

I’m happy to remove all references to Sonarcloud, if we’re not using it anymore.

File: README.md (hunk 2 above)
Comment (adunkman) [reply README.md:29]: Correct! They’re referenced here with a lock icon (🔒) to indicate as such — I think keeping the documentation together and linking out to private sections helps keep everyone on the same page. One example of this pattern is the [TTS Handbook](https://handbook.tts.gsa.gov/) which also adds lock icons to indicate when items are private.

File: .github/ISSUE_TEMPLATE/onboarding.md (hunk 1 above)
Comment (kkoskelin) [reply .github/ISSUE_TEMPLATE/onboarding.md]: Amendment: the removal of Sonarcloud will be completed after Batch 1 is merged.

File: docs/how-we-work.md [hunk 4]
```diff
@@ -1,97 +1,100 @@
...
+1.  Development team completes work — see _Definition of Done_ above
+2.  The product team verifies the functionality against acceptance criteria in a deployed instance
+3.  Development team creates pull request to the Court's staging environment
+4.  The code is reviewed by the Court for compliance with the contract's [performance and deliverable standards](https://github.com/ustaxcourt/case-management-rfq/blob/master/02_SOW.md#deliverables-and-performance-standards), and for general technical quality — see [Code Review Process](https://github.com/ustaxcourt/ef-cms/blob/staging/docs/CODE_REVIEW.md)
+5.  The development team responds to the Court’s code review feedback
+6.  Pull request merged to staging branch and deployed to the staging environment by the Court
+7.  User testing is performed
+8.  Court creates pull request to production branch and merges to update the production environment
```
Comment (JessicaMarine) [inline docs/how-we-work.md:73]: We might want to outline the two different Git flows: bugs vs. story work.

File: docs/how-we-work.md [hunk 5]
```diff
@@ -1,97 +1,100 @@
...
 
-6.  **Build atop a modern technology stack.** We use cloud-based hosting infrastructure, modular software frameworks, containerization, and automated testing to increase efficiency, allow the team to work effectively, and scale the service on demand.
+There are two basic meeting rhythms: daily design calls/standups and bi-weekly agile sprint rituals. All meetings are held via video teleconference. A telephone bridge is maintained as a backup method of connecting, but participants are strongly encouraged to join via video.
 
-7.  **Favor emergent architecture over "big up-front design."** Consistent with agile, we optimize for serving the emergent needs of users, instead of planning and executing an architecture that cannot respond to those changing needs.
+### Daily
 
-8.  **Deliver incremental value.** We deliver value to users as often as possible.
+- **Daily design sync** — this meeting is a time to solution functionality, demo in-progress work with the team and review new wireframes
+- **Daily standup**
```
Comment (JessicaMarine) [inline docs/how-we-work.md:22]: Might want to add something about our Monday & Friday DAWSON touch-base with Judge Buch and Stephanie.

Comment (mmarcotte) [reply docs/how-we-work.md:22]: ... a stake holder meeting

### Weekly

- **Stakeholder Meeting**  - on Mondays and Fridays, we review the current project status with stakeholders at the Court and discuss the schedule of large releases that need off-hours deployments.

File: docs/how-we-work.md (hunk 4 above)
Comment (mmarcotte) [reply docs/how-we-work.md:73]: For this, Pull requests of story work is a PR directly to the `staging` branch after testing has been conducted in the `migration` environment.

For Bugfixes, Pull requests are opened to the `test` environment from work based off of `staging`. If approved, a new PR is made to the `staging` branch and readied for production deploy.

**Comment by adunkman** [comment]
@mmarcotte PR updated!

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**